### ISSUE TITLE
[DOC] - Add alt text and mouseover comments

### ISSUE THREAD

**Issue by kcpevey** [nebari-dev/nebari-docs#104]
### Preliminary Checks

- [X] This issue is not a question, feature request, RFC, or anything other than a bug report. Please post those things in GitHub Discussions: https://github.com/nebari-dev/nebari/discussions

### Summary

[Original comment/source](https://github.com/nebari-dev/nebari-docs/pull/45#discussion_r907643807)

Add alt text to one place in the Getting started docs

### Steps to Resolve this Issue

add some alt text...

**Comment by trallard** [comment]
Just a note here - I am not really a fan of mouseovers so let's shelf that for now and focus on alt text

**Comment by kcpevey** [comment]
Yep, sounds good!

**Comment by pavithraes** [comment]
We can have a company-wide sprint to do this (maybe during a QShare!).

**Comment by trallard** [comment]
Coming back to this - shall we plan a sprint?

**Comment by pavithraes** [comment]
Yes! [We may be able to start this in today's QShare](https://quansight.slack.com/archives/CAERL8LSE/p1668172415492679). If there isn't enough time, we can complete the work in a dedicated sprint. :)
